admin/services/school.service.js

```diff
@@ -40,9 +41,21 @@ const schoolService = {
   },
 
   /**
-   *
+   * @typedef {object} schoolRecord
+   * @property {number } id
+   * @property {string} name,
+   * @property {number} leaCode,
+   * @property {number} estabCode,
+   * @property {number} dfeNumber,
+   * @property {number} urn,
+   * @property {string} urlSlug,
+   * @property {number} numberOfPupils
+   * @property {array} audits
```

Comment:
It would be nice to know what this array is actually of?

Reply:
thats actually no longer part of the definition, have removed.